UKHomeOffice/sgmr-service
# NMSW-42 Departure/Arrival port hint text wording

### AC / Description of changes

GIVEN I am on the departure page of the submit voyage form
WHEN I view the port quest
THEN I see the hint text of 'If you are planning a voyage between two UK ports or between two international ports, please do not submit a plan.'

GIVEN I am on the arrival page of the submit voyage form
WHEN I view the port quest
THEN I see the hint text of 'If you are planning a voyage between two UK ports or between two international ports, please do not submit a plan.'

### To Test

- Run app locally
- Go to localhost:8080
- Create a voyage plan
- > On **departure** details you should see the hint text on the port field read 'If you are planning a voyage between two UK ports or between two international ports, please do not submit a plan.'
- Complete the departure page
- Click continue
- > On **arrival** details you should see the hint text on the port field read 'If you are planning a voyage between two UK ports or between two international ports, please do not submit a plan.'

### Notes

---

**Remember to merge to feature branches not `master`.**
